Formula/m/mysql@8.0.rb

```diff
@@ -102,7 +102,10 @@ def install
     system "cmake", "--install", "build"
 
     cd prefix/"mysql-test" do
-      system "./mysql-test-run.pl", "status", "--vardir=#{buildpath}/mysql-test-vardir"
+      system "./mysql-test-run.pl", "check", "--debug", "--vardir=#{buildpath}/mysql-test-vardir"
```

Comment:
```suggestion
      system "./mysql-test-run.pl", "check", "--vardir=#{buildpath}/mysql-test-vardir"
```

Reply:
We built with debug enabled to see if there was any difference in the logging, but It looks like the flag wasn't removed when debug was disabled.

Reply:
Yeah, we're not building with debug anymore so the flag needs to be removed here too.